README.md

```diff
@@ -265,6 +265,31 @@ Note that some undocumented records are used for internal state or testing. They
    </tr>
 </table>
 
+### Internal/Diagnostic Log Record Types ###
+The .gt3x file also contains record types utilized internally by ActiGraph engineers for diagnostic purposes. Records with these types can be ignored (or skipped over). The following are record types utilized for diagnostic purposes:
+
+<table>

```

Comment:
Could use [markdown table](https://www.markdownguide.org/extended-syntax/#tables) here, but this works.